### DISCUSSION TITLE
Dose scoring ausgab object output units and normalization?

### DISCUSSION THREAD

**Discussion by kstcmcjw** [nrc-cnrc/EGSnrc#890] [Questions]
Dear every guider, thanks for you answer me so many questions! Now i face a dose question, it is relate to the program, I creat a 1cm³ glass-chamber in order to instead of a ionization chamber, my thought is that the old principle:

 "exposure, X" was defined 0.001293g of secondary electrons released from air under 1 Roentgen X-ray irradiation. (standard condition, 1cm³ of air mass),  so i created the glass spheres as a glass_chambers, aim to monitor the sphere region dose.

![image](https://user-images.githubusercontent.com/96934217/177559008-be2f55e0-0329-4240-8020-b2937047c8e6.png)

As the follow shows:

---------------------------------------------------------

 :start ausgab object:
      library = egs_dose_scoring
      name = kongqi_chambers
      medium dose  = no
      region dose  = yes
      volume = 1
      dose regions = glass_chambers
 :stop ausgab object:

-----------------------------------------------------------

Then, the EGSnrc_gui software run the simulation reslut:

![image](https://user-images.githubusercontent.com/96934217/177566142-187d35f5-c823-4431-91f9-9450bd5cef22.png)

There are four questions puzzled me:

the first is the "medium dose" and the "region dose", I think the medium dose is the barrier absorb dose, it means that the defined barrier which stop the ray dirction in the  diffuse tracks, such as a volume of concrete, and the region dose is the area which I am care about it, in my experiment, the glass_chambers is my care area, so i want the glass-chamber's dose, is it right?

the second is the reslut, in the simulation, the region dose is D/[Gy*cm2], what is the region dosimetry (per fluence) and Gy*cm2 stand for? Is it the energy fluence? I can't find this definition, the energy fluence is the J per ㎡,  is this the "dose fluence"?

the third is, if  D/[Gy*cm2] is the "dose fluence",  J/㎡， how to translate it to the absorb dose of the 1cm³ of my glass spheres?

the fourth, in order to compare with different thick of lead plate, concrete, brick and other materials, i replaced different of lead, 1mm, 2mm, 3mm, 5mm, but the dose output reslut is 3.5145e-007±0.194%, then i replace the concrete, water, as if I seen a ghost, the dose output reslut is 3.5145e-007±0.194% still! How can it???
The program as follow:

--------------------------------------------------------
###1.RUN CONTROL
:start run control:
    ncase = 5e6
:stop run control:

###2.GEOMETRY
:start geometry definition:
  :start geometry: 
    name = cd_planes
    library = egs_planes
    type = EGS_Zplanes
    positions = -80 -0.5 0 500
  :stop geometry:

  :start geometry:
    name = air
    library = egs_box
    box size = 100 100 600
        :start media input:
            media = air
        :stop media input:
  :stop geometry:
  simulation geometry = air

  :start geometry:
    name = barrier_lead
    library = egs_box
    box size = 100 100 10
        :start media input:
            media = lead
        :stop media input:
  :stop geometry:
  simulation geometry = barrier_lead

  :start geometry:
    name = chamber
    library = egs_spheres
    library = egs_planes
    type = EGS_Zplanes
    midpoint = 0 0 -30.5
    radii = 0.6204 0.7204
    :start media input:
        media = air glass
        set medium = 1 1
        set medium = 2 2
    :stop media input:
  :stop geometry:
  simulation geometry = chamber

    :start geometry:
        library  = egs_genvelope
        name = air_chambers
        base geometry = air
        inscribed geometries = chamber 
    :stop geometry:
    simulation geometry = air_chambers
  
  :start geometry:
    name = protection
    library = egs_cdgeometry
    base geometry =cd_planes
    set geometry = 2 air
    set geometry = 1 barrier_lead
    set geometry = 0 air_chambers

  :stop geometry:
  simulation geometry = protection

:stop geometry definition:

###3.MEDIA
:start media definition: 
  ae = 0.521 # lowest energy for electrons (kinetic+0.511)
  ap = 0.01 # lowest energy for photons (kinetic)
  ue = 50.511 # maximum energy for electrons (kinetic+0.511)
  up = 50 # maximum energy for photons (kinetic)
 
  :start lead:
      density correction file = water_liquid
  :stop lead:

  :start air:
      density correction file = air_dry_nearsealevel
  :stop air:

  :start glass:
      density correction file = glass_borosilicate__pyrex__corning7740
  :stop glass:

:stop media definition:

###4.SOURCE
:start source definition:
    :start source:
        library = egs_collimated_source
        name = tube_source
        :start source shape:
            type = point
            position = 0 0 200
        :stop source shape:
        :start target shape:
            library = egs_rectangle
            rectangle = -20 -20 20 20
        :stop target shape:
            distance = 200
            charge = 0
        :start spectrum:
            type = monoenergetic
            energy = 0.12
        :stop spectrum:
    :stop source:
    simulation source = tube_source
:stop source definition:

###5.VIEWER CONTROL
:start view control:
  set color = lead 128 128 128 255
  set color = air 0 200 200 200
  set color = glass 255 0 255 255
 
:stop view control:

###6.AUSGAB OBJECTS
:start ausgab object definition: 
    :start ausgab object:
      name = tracks
      library = egs_track_scoring
    :stop ausgab object:

  :start ausgab object:
      library = egs_dose_scoring
      name = air_chambers
      medium dose  = no
      region dose  = yes
      volume = 1
      dose regions = air_chambers
  :stop ausgab object:
:stop ausgab object definition:

------------------------------------------------------

the reslut, different shielding material and thickness calculated the same reslut???
![image](https://user-images.githubusercontent.com/96934217/177577792-bab2214a-ada9-4bef-aaf7-de6d24a875b8.png)

Thanks for your attention, have a nice day!

**Comment by rtownson** [comment]
```dose regions = air_chambers```

This should be a list of region numbers, as seen in egs_view. Currently, the dose is just reported to region 0, which I believe is the air box surrounding anything (that's why it doesn't change). 

Dose is reported per source fluence, which for the collimated source you can better understand by working through these slides:
https://nrc-cnrc.slides.com/ftessier/egsnrc-collimated-source/fullscreen

If you set `distance` to be equal to 1/sqrt(4*pi), then you can just take the fluence to equal the number of source particles, so the dose will be Gy per source particle.

By the way, the getting-started manual explains many of these questions: https://nrc-cnrc.github.io/EGSnrc/

**Reply by kstcmcjw** [reply]
Er, thank you for your answer, I am a clumsy and stupid guy. After reading the EGSnrc introductory manual, I couldn't understand so many knowledge points, so I have so many problems. I am very grateful to you and everyone from the bottom of my heart. 
https://nrc-cnrc.slides.com/ftessier/egsnrc-collimated-source/fullscreen

This online PPT is very nice! I will study it, by the way, how many knowledge of EGSnrc that the user_guide getting_start PDF hasn't contain? Such as the https://nrc-cnrc.slides.com/ftessier/egsnrc-collimated-source/fullscreen and the density_make online?

**Reply by kstcmcjw** [reply]
This should be a list of region numbers, as seen in egs_view. Currently, the dose is just reported to region 0, which I believe is the air box surrounding anything (that's why it doesn't change).

but how can I get the glass-chamber dose? 
in the chapter 4.4 , it seems the medium dose, it report the per particle, Whether the medium dose report the per particle, the region dose of collimated source report the fluence?
In my program, I should exchange the dose regions to the glass number from EGS_view, I see the collimated source fluence, a complex calculate, then the fluence should be calculate with the  equation?

![image](https://user-images.githubusercontent.com/96934217/177688058-53b22cd2-c4bb-490f-a4b6-a77fdfcac9ba.png)

**Reply by rtownson** [reply]
The dose is per particle because of the type of source - it is always normalized per fluence, but the "fluence" for that type of source is defined as the number of source particles. The collimated source is one of the few that is defined differently.

When you ask for dose per medium, it just adds up the dose to all of the regions corresponding to that medium. The normalization is the same.

**Reply by kstcmcjw** [reply]
Thank you very much, you help me so many times, I think the collimated source is widely used in the medical exposure, but it seems difficult to calculate from the "fluence" to the "region dose".

I will try my poor program again! 

Thank you, have a nice weekend!

**Reply by rtownson** [reply]
It's very easy, for the collimated source just set `distance = 0.28209` (which is $\approx 1/\sqrt{4\pi}$), and then the fluence=number of source particles, like any other source.

**Reply by kstcmcjw** [reply]
Thank you very much, i am so sorry for reply so late, I am working hard and busy with construction of department of nuclear medicine these days, thanks very much!
